src/Workspaces/Core/Portable/Workspace/WorkspaceDiagnostic.cs

```diff
@@ -1,10 +1,11 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System.Diagnostics;
+using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis
 {
-    [DebuggerDisplay("{GetDebuggerDisplay(),nq}")]
+    [DebuggerDisplay("{ToString(),nq}")]
```

Comment:
This is now no longer needed, the debugger automatically calls ToString. You might be misremembering, when Roslyn expression evaluator broke this - but I've confirmed it's fixed in RTM.